README.md

```diff
@@ -1,35 +1,72 @@
 
-## To Use Yourself
+## Usage
+#### 1. Get a [buttondown](https://buttondown.email/) account and api key.
 
-### Get a [buttondown](https://buttondown.email/) account and api key.
-
-![api_key in buttondown-settings](assets/api_buttondown-settings.jpg)
-
-### Create a template of this repo
+Visit the [buttondown](https://buttondown.email/) website and create an account. Once you have an account. Setup your newsletter. 
+#### 2. Create a template of this repo
 
 There is a create template button on the top right of this repo. Click it and follow the instructions to create a template of this repo in your own account.
 
-### Clone the template
+#### 3. Clone the template
 
 Clone the template to your local machine or cloud environment.
 
-### Create a virtual environment and install dependencies
+#### 4. Create a virtual environment and install dependencies
 
 ```bash
 python -m venv venv
 source venv/bin/activate
 python -m pip install -r requirements.txt
 ```
 
-### Run the `publish_show` with the issue number
+#### 5. Store your api key in environment variables
+
+You'll need some information to pass to the script. Most of the information can be passed as environment variables or manually.
+
+#### Required Values
+You will need the following information:
+
+- `BUTTONDOWN_API_KEY` - Your buttondown api key. You can find this in your [buttondown](https://buttondown.email/) account settings.
+
+![api_key in buttondown-settings](assets/api_buttondown-settings.jpg)
+
+- `GITHUB_ACCOUNT` - Your github account name. This is the name of the account that owns the repo you want to pull issues from.
+- `GITHUB_REPO` - The name of the repo you want to pull issues from.
+
+### Optional Values
+
+- `GITHUB_API_TOKEN` - Your github api token. You can create one in your GitHub account settings in the Developer Settings section. If you have one, you can use it to increase the rate limit of the GitHub API. You can read more about the GitHub API rate limit [here](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting).
+    
+![Github Access Token](assets/personal_access_token.png)
+
+Run the `publish_show.py` with the issue number
 
 ```bash
-python publish_show.py --issue_number=1
+export BUTTONDOWN_API_KEY=<your_api_key>
+export GITHUB_ACCOUNT=<your_github_account>
+export GITHUB_REPO=<your_github_repo>
+export GITHUB_API_TOKEN=<your_github_api_token>
+
+python publish_show.py 1 # replace 1 with the issue number you want to publish
+
 ```
+
+Alternatively, you can pass the values as arguments to the script.
+
+```bash
+python publish_show.py 1 \ 
+--buttondown_api_key=<your_api_key> \ 
+--github_account=<your_github_account> \
+--github_repo=<your_github_repo> \
+--github_api=<your_github_api_token>
+```
+
+You can also call the script with the `--help` flag to see all the options.
+![](assets/publish_show_help.png)
```

Comment:
I'm guessing you want the color formatting in the image you've included here, but can you also provide a text version directly in the readme and add alt text to the image?

Reply:
Alt Text absolutely!

The text version of the output may not format correctly, but I can try.

Reply:
Yeah the output was a garbled mess that said I did add the full command python `publish_newsletter.py --help`